src/lib/util/Security/AntiRaid.ts

```diff
@@ -48,7 +49,7 @@ export class AntiRaid extends Collection<string, AntiRaidEntry> {
 	public create(id: string) {
 		const rateLimit = { id, time: Date.now() + 20000 };
 		this.set(id, rateLimit);
-		if (!this._sweepInterval) this._sweepInterval = (setInterval(this.sweep.bind(this), 30000) as unknown) as NodeJS.Timeout;
+		if (!this._sweepInterval) this._sweepInterval = cast<NodeJS.Timeout>(setInterval(this.sweep.bind(this), 30000));
```

Comment:
This cast feels..off and not needed

Reply:
setInterval's return type somehow is `number`...

Reply:
Sounds like a nodejs conflict with DOM types 🤷‍♂️.. understandable tho